src/functions-reference/matrix_operations.Rmd

```diff
@@ -1237,66 +1237,270 @@ The cumulative sum of v
+`matrix` **`gp_exp_quad_cov`**`(real[] x, real sigma, real length_scale)`<br>\newline
 
-`matrix` **`cov_exp_quad`**`(vectors x, real alpha, real rho)`<br>\newline
-The covariance matrix with an exponentiated quadratic kernel of x.
+Gaussian process with squared exponential kernel in one dimension.
 
-<!-- matrix; cov_exp_quad; (real[] x, real alpha, real rho); -->
-\index{{\tt \bfseries cov\_exp\_quad }!{\tt (real[] x, real alpha, real rho): matrix}|hyperpage}
+\index{{\tt \bfseries gp\_exp\_quad\_cov }!{\tt (real[] x1, real[] x2, real sigma, real length_scale): matrix}|hyperpage}
 
-`matrix` **`cov_exp_quad`**`(real[] x, real alpha, real rho)`<br>\newline
-The covariance matrix with an exponentiated quadratic kernel of x.
+`matrix` **`gp_exp_quad_cov`**`(real[] x1, real[] x2, real sigma, real length_scale)`<br>\newline
 
-<!-- matrix; cov_exp_quad; (row_vectors x1, row_vectors x2, real alpha, real rho); -->
-\index{{\tt \bfseries cov\_exp\_quad }!{\tt (row\_vectors x1, row\_vectors x2, real alpha, real rho): matrix}|hyperpage}
+Gaussian process with squared exponential kernel in one dimension between `x1`
+and `x2`.
 
-`matrix` **`cov_exp_quad`**`(row_vectors x1, row_vectors x2, real alpha, real rho)`<br>\newline
-The covariance matrix with an exponentiated quadratic kernel of x1 and
-x2.
+\index{{\tt \bfseries gp\_exp\_quad\_cov }!{\tt (vector[] x, real sigma, real length_scale): matrix}|hyperpage}
 
-<!-- matrix; cov_exp_quad; (vectors x1, vectors x2, real alpha, real rho); -->
-\index{{\tt \bfseries cov\_exp\_quad }!{\tt (vectors x1, vectors x2, real alpha, real rho): matrix}|hyperpage}
+`matrix` **`gp_exp_quad_cov`**`(vector[] x, real sigma, real length_scale)`<br>\newline
 
-`matrix` **`cov_exp_quad`**`(vectors x1, vectors x2, real alpha, real rho)`<br>\newline
-The covariance matrix with an exponentiated quadratic kernel of x1 and
-x2.
+Gaussian process with squared exponential kernel in multiple dimensions.
 
-<!-- matrix; cov_exp_quad; (real[] x1, real[] x2, real alpha, real rho); -->
-\index{{\tt \bfseries cov\_exp\_quad }!{\tt (real[] x1, real[] x2, real alpha, real rho): matrix}|hyperpage}
+\index{{\tt \bfseries gp\_exp\_quad\_cov }!{\tt (vector[] x, real sigma, real[] length_scale): matrix}|hyperpage}
 
-`matrix` **`cov_exp_quad`**`(real[] x1, real[] x2, real alpha, real rho)`<br>\newline
-The covariance matrix with an exponentiated quadratic kernel of x1 and
-x2.
+`matrix` **`gp_exp_quad_cov`**`(vector[] x, real sigma, real[] length_scale)`<br>\newline
+
+Gaussian process with squared exponential kernel in multiple dimensions with a
+per-dimensional length scale.
+
+\index{{\tt \bfseries gp\_exp\_quad\_cov }!{\tt (vector[] x1, vector[] x2, real sigma, real length_scale): matrix}|hyperpage}
+
+`matrix` **`gp_exp_quad_cov`**`(vector[] x1, vector[] x2, real sigma, real length_scale)`<br>\newline
+
+Gaussian process with squared exponential kernel in multiple dimensions between
```

Comment:
Something like: computes the cross covariance between two sets of data points x1, x2. Number of dimensions must be the same, as well.